### DISCUSSION TITLE
Feature request - Screen Thumbnail preview

### DISCUSSION THREAD

**Discussion by cedric-marcoux** [Ylianst/MeshCentral#6478] [Ideas]
Hello,

i was using ConnectWise Screenconnect before and they have a great feature to know if user is busy or not, thus was a thumbnail preview of the screen of the user. It could be great to have a new tab on device that show as similar. This feature is usefull when you wait for a user to idle before doing something on schedule without connecting every 5' de see if the computer was free or not.
It was looking like that on connectwise screenconnect:
![image](https://github.com/user-attachments/assets/47c01f7f-f19e-4205-978c-3db2a849c2e8)

I'm not programmer but i think fetching a jpeg every minute should not be a difficult feature to implement.

Thanks for help and thanks for this great software

**Comment by si458** [comment]
There are already open requests for this feature, and it's on the to-do list as it's not very simple to implement 

Some businesses have policies in place that restrict viewing some desktops without end user consent!

So we would have to add an option in for this feature so it can be turned on (and turned off by default)

**Reply by Darkpriest258** [reply]
do you still use screenconnect??

**Comment by cedric-marcoux** [comment]
Hello,

it could be combined with the existing "**User consent**" screen settings, kind of "Allow preview" checkbox...
![image](https://github.com/user-attachments/assets/c646914c-1b71-4ded-b6c9-a3065578f3a0)

**Reply by si458** [reply]
my idea would be a setting in config.json then it's per domain or globally

Also taking snapshots of the desktop uses the kvm module which can take a few seconds to process, but leave this with me :)
